src/components/Menu.tsx

```diff
@@ -171,44 +172,44 @@ const Menu = (props) => {
               rerender={renderFormKeys}
             />
           </div>
-          <div id="routingResults" hidden={!props.duration}>
-            <TimerIcon
-              htmlColor={"gray"}
-              fontSize={"small"}
-              sx={{ marginLeft: "5px" }}
-            />
-            <span style={{ margin: "5px" }}>{duration}</span>
-            <HeightIcon
-              htmlColor={"gray"}
-              sx={{ transform: "rotate(90deg)", marginLeft: "5px" }}
-            />
-            <span style={{ margin: "5px" }}>{distance} km</span>
-            <span
-              className="elevation-details-trigger"
-              style={{ marginLeft: "5px" }}
-              onMouseOver={() => setShowElevationPopup(true)}
-            >
-              <ExpandIcon htmlColor={"white"} sx={{ marginRight: "5px" }} />
-              {elevation} m
-            </span>
-          </div>
+          {props.duration !== null && props.distance !== null && (
```

Comment:
Is this a better approach considering performance? I guess, the div itself is not evaluated this way.

Reply:
The extra null check is just to make TypeScript happy. Previously, if `props.duration` was set, but `props.distance` was null then the div would still render, but TypeScript would complain about the usage of `props.duration` inside that block, and preventing a potential bug. Would have been a very small bug though, but still nice to be certain about the types of things.